py/janitor/worker_creds.py

```diff
@@ -41,13 +41,13 @@ async def check_worker_creds(db, request: web.Request) -> Optional[str]:
     auth_header = request.headers.get(aiohttp.hdrs.AUTHORIZATION)
     if not auth_header:
         raise web.HTTPUnauthorized(
-            text="worker login required",
+            text="worker login required (missing auth_header)",
```

Comment:
auth_header is an internal variable, meaningless to the user

Reply:
My thinking for this. I'm currently trying:

```
root@testing:/janitor# ./target/release/janitor-worker --base-url http://janitor.dev.local --credentials credentials.json
[2025-03-04T16:51:04Z INFO  janitor_worker] listening on 0.0.0.0:46849
[2025-03-04T16:51:10Z ERROR janitor_worker] failed to get assignment: worker login required
root@testing:/janitor#
```

I'm not sure why/where its currently failing.

I'm not sure if its due to the header missing or creds are incorrect. Its adding extra steps into troubleshooting.

Reply:
okay - dropping

Reply:
Done